### PR TITLE
Fix UIKit Case Study warning

### PR DESCRIPTION
**Pull request by stephencelis**
Noticed a deprecation warning for using an old TCA API. I'm not 100% satisfied with the fix yet...but it's probably better than leaving as is.

### CODE REVIEW

File: Examples/CaseStudies/tvOSCaseStudies/FocusView.swift [hunk 1]
```diff
@@ -48,7 +48,13 @@ let focusReducer = Reducer<FocusState, FocusAction, FocusEnvironment> {
...
 
           HStack(spacing: 40) {
-            ForEach(1..<11) { index in
+            ForEach(1..<6) { index in
+              Button(numbers[index]) {}
+                .prefersDefaultFocus(viewStore.currentFocus == index, in: self.namespace)
+            }
+          }
+          HStack(spacing: 40) {
+            ForEach(6..<11) { index in
```
Comment (stephencelis) [inline Examples/CaseStudies/tvOSCaseStudies/FocusView.swift:57]: Unrelated fix to render the tvOS focus study more cleanly: 
image

**Review by mbrandonw — APPROVED** [review]
